packages/components/src/utils/useShortcut.test.tsx

```diff
@@ -0,0 +1,62 @@
+import { useShortcut } from '@/utils/useShortcut';
+import { fireEvent, render } from '@testing-library/react';
+
+type TestComponentProps = {
+  shortcutKey: string;
+  callback: () => void;
+  modifiers?: { ctrl?: boolean; alt?: boolean; shift?: boolean };
+};
+
+const TestComponent = ({ shortcutKey: key, callback, modifiers }: TestComponentProps) => {
+  useShortcut(key, callback, modifiers);
+  return <div>Test Component</div>;
+};
+
+test('default', () => {
+  let shortcutTriggered = false;
+  render(<TestComponent shortcutKey={'a'} callback={() => (shortcutTriggered = true)} />);
+
+  fireEvent.keyDown(document, { key: 'a', ctrlKey: true, shiftKey: true });
```

Comment:
Isn't fireEvent deprecated and userEvent should be used?

Reply:
Not deprecated it seems but I just found this remark [in the doc](https://testing-library.com/docs/dom-testing-library/api-events/):
> Most projects have a few use cases for fireEvent, but the majority of the time you should probably use [@testing-library/user-event](https://testing-library.com/docs/user-event/intro).

I'll check.